### ISSUE TITLE
Not working on Win10 x64

### ISSUE THREAD

**Issue by barabasz** [quinton-ashley/cemu-no-gyro#2]
Windows 10.0.17763.253 x64
CEMU 1.15.0e

```
Error: Failed to lookup view "pug/client" in views directory "./views"
    at Function.render (C:\cemu-no-gyro\resources\app.asar\node_modules\express\lib\application.js:580:17)
    at ServerResponse.render (C:\cemu-no-gyro\resources\app.asar\node_modules\express\lib\response.js:1008:7)
    at C:\cemu-no-gyro\resources\app.asar\views\js\index.js:214:7
    at Layer.handle [as handle_request] (C:\cemu-no-gyro\resources\app.asar\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\cemu-no-gyro\resources\app.asar\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\cemu-no-gyro\resources\app.asar\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\cemu-no-gyro\resources\app.asar\node_modules\express\lib\router\layer.js:95:5)
    at C:\cemu-no-gyro\resources\app.asar\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\cemu-no-gyro\resources\app.asar\node_modules\express\lib\router\index.js:335:12)
    at next (C:\cemu-no-gyro\resources\app.asar\node_modules\express\lib\router\index.js:275:10)
```

**Comment by SomeoneIsWorking** [comment]
Runs successfully if you download the source and run it with npm i && npm run
Probably was build incorrectly
But it is very very very wonky, I couldn't get it to work properly.

**Comment by quinton-ashley** [comment]
Try the latest build!  Let me know if it works for you.

**Comment by quinton-ashley** [comment]
You're right it doesn't work for some reason in the builds.  I'll check it out.

**Comment by quinton-ashley** [comment]
Fixed in v1.0.15
